shared/toks-components/src/constants/imageUrl.ts

```diff
@@ -0,0 +1,6 @@
+export const imageUrl = {
+  baseToks: 'https://asset.tokstudy.com/toks-emoji/ic-base-profile.png',
+  baseToksGrey: 'https://toks-web-assets.s3.amazonaws.com/ic-base-profile-grey.png',
+  baseKakao: 'http://k.kakaocdn.net/dn/dpk9l1/btqmGhA2lKL/Oz0wDuJn1YV2DIn92f6DVK/img_640x640.jpg',
+  logo: 'https://asset.tokstudy.com/logo.png',
+};
```

Comment:
상수값 네이밍으로 쓰면 좋을 것 같습니다

```suggestion
export const IMAGE_URL = {
  BASE_TOKS: 'https://asset.tokstudy.com/toks-emoji/ic-base-profile.png',
  BASE_TOKS_GREY: 'https://toks-web-assets.s3.amazonaws.com/ic-base-profile-grey.png',
  BASE_KAKAO: 'http://k.kakaocdn.net/dn/dpk9l1/btqmGhA2lKL/Oz0wDuJn1YV2DIn92f6DVK/img_640x640.jpg',
  LOGO: 'https://asset.tokstudy.com/logo.png',
};
```

Reply:
🙏